source/layouts/application.erb

```diff
@@ -87,10 +87,8 @@
             <%= site_nav_link('Thoughts', '/thoughts') %>
             <%= site_nav_link('People', '/people') %>
             <%= site_nav_link('Blog', '/blog') %>
-            <%= site_nav_link('News', '/news') %>
             <%# Removed temporarily to test change in user flow https://www.pivotaltracker.com/story/show/137786451 %>
             <%#= site_nav_link('Spinouts', '/spinouts') %>
```

Comment:
Unrelated to your changes but while we're here can we remove these two commented lines please? Spinouts is now in the footer isn't it... I don't think we need these comments lying around